source/includes/_dataset_security.md

```diff
@@ -0,0 +1,285 @@
+# Dataset security
+
+The dataset security is defined through 3 variables:
+
+* a global accessibility policy
+* specific rulesets for users and groups
+* a default ruleset that applies to anybody else
+
+### Restricted datasets
```

Comment:
you could add a intermediate title with 'global accessibility policy' to explain that the restricted/unrestricted datasets is defined by this policy